galsim/__init__.py

```diff
@@ -147,6 +148,7 @@
 
 # Deprecation warning class
 from .deprecated import GalSimDeprecationWarning
+from .deprecated import optics as optics
```

Comment:
I'm confused about the `as optics` bit.  Seems redundant.

Also, isn't this superseded by the below `from . import optics'?